crates/prql-compiler/src/tests/test.rs

```diff
@@ -3402,6 +3402,16 @@ fn test_header_target_error() {
     "###);
 }
 
+#[test]
+fn prql_version() {
```

Comment:
@Hulxv this is the test I added

Reply:
And now it passes — I ran `cargo insta test --accept` and we can see the result now...